db/seeds.rb

```diff
@@ -5,3 +5,15 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+require "nokogiri"
+require "open-uri"
+
+doc = Nokogiri::HTML(open("http://study-english.info/vocabulary-questions.php"))
+
+doc.css("tr").each do |tr|
+  Card.create(
+    original_text: tr.css("td[bgcolor='#FFFFDD']").text,
+    translated_text: tr.css("td[bgcolor='#FFFFDD'] + td").text,
+    review_date: (DateTime.now.to_date + 3)
```

Comment:
Do not use `DateTime.now` without zone. Use one of `Time.zone.now`, `DateTime.now.current`, `DateTime.now.in_time_zone`, `DateTime.now.utc`, `DateTime.now.getlocal`, `DateTime.now.iso8601`, `DateTime.now.jisx0301`, `DateTime.now.rfc3339`, `DateTime.now.to_i`, `DateTime.now.to_f` instead.